addons/website/static/src/snippets/s_card/000.scss

```diff
@@ -47,12 +46,4 @@
             }
         }
     }
-
-    .o_card_img_adjust_h .o_card_img {
-        object-position: var(--card-img-ratio-align) 50%;
-    }
-
-    .o_card_img_adjust_v .o_card_img {
-        object-position: 50% var(--card-img-ratio-align);
-    }
```

Comment:
You can not remove those properties; if you do so, you will break existing snippets style ;)